### ISSUE TITLE
Regression during retraining

### ISSUE THREAD

**Issue by avraham1** [ultralytics/ultralytics#19962]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

I trained YOLO11cls on two classes.  
- **Class A:** 8K training images and 4K test images.  
- **Class B:** 20K training images and 10K test images.  

In the first training session, I ran **200 epochs**, and the final epoch reached a **loss function value of 0.18**.  
```
model = YOLO("yolo11n-cls.pt") 
results = model.train(data="images", epochs=200, imgsz=256)
```
Results.csv:
```
epoch,time,train/loss,metrics/accuracy_top1,metrics/accuracy_top5,val/loss,lr/pg0,lr/pg1,lr/pg2
1,179.484,0.42652,0.84315,1,0.40961,0.00333148,0.00333148,0.00333148
200,28714.6,0.18747,0.8858,1,0.30626,0.0001495,0.0001495,0.0001495

```

Then, I ran **another 400 epochs on the same dataset**, but what happened was that **in the very first epoch, the loss was already at 0.19**, and from that point onward, it **increased to 0.2–0.3**.  
```
model = YOLO("model\\runs\\classify\\train\\weights\\last.pt") 
results = model.train(data="images", epochs=400, imgsz=256)
```
```
epoch,time,train/loss,metrics/accuracy_top1,metrics/accuracy_top5,val/loss,lr/pg0,lr/pg1,lr/pg2
1,147.596,0.19665,0.88219,1,0.32262,0.00333148,0.00333148,0.00333148
41,6009.89,0.33643,0.88107,1,0.29097,0.00901,0.00901,0.00901

```

1. Could it be that the model **forgot what it had already learned in the first training session**?  
2. Why **doesn't it continue from where it left off in the first training session**?

**Comment by Y-T-G** [comment]
Because it starts again with a higer learning rate.

https://www.reddit.com/r/Ultralytics/comments/1eolwl8/the_correct_way_to_train_from_a_previously/
